pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/SyntheticStageGraphListener.java

```diff
@@ -79,28 +80,23 @@ private void attachTag(FlowNode currentNode, String syntheticContext) {
         }
     }
 
-    public static class GraphListenerAction extends InvisibleAction implements RunAction2 {
```

Comment:
Note that this may trigger some `OldDataMonitor` warnings. Harmless I think, though if you want to avoid it you can keep the action around, but deprecated and a no-op.